python/ray/tests/test_runtime_env_complicated.py

```diff
@@ -658,36 +618,33 @@ def assert_tasks_finish_quickly(total_sleep_s=0.1):
 
 
 @pytest.mark.skipif(
-    os.environ.get("CI") is None,
-    reason="This test is only run on CI because it uses the built Ray wheel.")
-@pytest.mark.skipif(
-    sys.platform != "linux", reason="This test is only run on Buildkite.")
+    os.environ.get("CI") and sys.platform != "linux",
+    reason="This test is only run on linux CI machines.")
 def test_simultaneous_install(shutdown_only):
     """Test that two envs can be installed without affecting each other."""
     ray.init()
 
     @ray.remote
-    class TensorflowWorker:
-        def __init__(self):
-            import tensorflow as tf
-            self.version = tf.__version__
+    class VersionWorker:
+        def __init__(self, key):
+            self.key = key
 
-        def get_version(self):
-            return self.version
+        def get(self):
+            return self.key
 
     # Before we used a global lock on conda installs, these two envs would be
     # installed concurrently, leading to errors:
     # https://github.com/ray-project/ray/issues/17086
     # Now we use a global lock, so the envs are installed sequentially.
-    tf1 = TensorflowWorker.options(runtime_env={
-        "pip": ["tensorflow==2.4.2"]
-    }).remote()
-    tf2 = TensorflowWorker.options(runtime_env={
-        "pip": ["tensorflow==2.5.0"]
-    }).remote()
-
-    assert ray.get(tf1.get_version.remote()) == "2.4.2"
-    assert ray.get(tf2.get_version.remote()) == "2.5.0"
+    worker_1 = VersionWorker.options(runtime_env={
+        "pip": ["pip_install_test==0.5"]
+    }).remote(key=1)
+    worker_2 = VersionWorker.options(runtime_env={
+        "pip": ["pip_install_test==0.4"]
+    }).remote(key=2)
+
+    assert ray.get(worker_1.get.remote()) == 1
+    assert ray.get(worker_2.get.remote()) == 2
```

Comment:
Is it worth putting requests==2.2.0 and requests==2.3.0 in `pip` and having the workers print out the version?  That way we can also ensure the workers have the correct runtime env.